@here/olp-sdk-dataservice-read/lib/client/ArtifactClient.ts

```diff
@@ -64,11 +64,13 @@ export class ArtifactClient {
             "artifact",
             this.apiVersion,
             this.settings
-        ).catch(error => Promise.reject(error));
+        ).catch(error =>
+            Promise.reject(new HttpError(error.status, error.message))
```

Comment:
we don't need that, because error is already a HttpError instance

Reply:
fixed